.cicdtemplate/.bitrise/bitrise.yml

```diff
@@ -0,0 +1,214 @@
+project_type: android
+trigger_map:
+  - push_branch: develop
+    workflow: develop
+
+  - push_branch: release/*
+    workflow: release
+
+  - push_branch: master
+    workflow: master
+
+  - push_branch: '*'
+    workflow: primary
+
+  - pull_request_source_branch: '*'
+    workflow: primary
+
+# Work-flows definition
+workflows:
+  # Workflow on * branch
+  primary:
+    steps:
+      - activate-ssh-key@4:
+          run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+
+      - git-clone@4: {}
+
+      - cache-pull@2: {}
+
+      - install-missing-android-tools@2:
+          inputs:
+            - gradlew_path: "$PROJECT_LOCATION/gradlew"
+
+      # Execute Lint check
+      - android-lint@0:
+          inputs:
+            - project_location: "$PROJECT_LOCATION"
+
+      # Execute Detekt
+      - gradle-runner@1:
+          inputs:
+            - gradlew_path: "$PROJECT_LOCATION/gradlew"
+            - gradle_task: detekt
+          title: Detekt
+
+      # Execute UnitTest
+      - android-unit-test@1:
+          inputs:
+            - arguments: jacocoTestReport
+            - variant: "$VARIANT"
+            - module: "$MODULE"
+            - report_path_pattern: '*build/reports'
+            - project_location: "$PROJECT_LOCATION"
+
+      # Upload jacocoTestReport to Apps & Artifacts for later downloading
+      - deploy-to-bitrise-io@1:
+          inputs:
+            - deploy_path: "$PROJECT_LOCATION/$MODULE/build/reports/jacoco/jacocoTestReport/html"
+            - is_enable_public_page: 'false'
+            - is_compress: 'true'
+            - notify_user_groups: none
+
+  # Workflow on develop branch
+  develop:
+    steps:
+      - activate-ssh-key@4:
+          run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+
+      - git-clone@4: {}
+
+      - cache-pull@2: {}
+
+      # Download Keystore file
+      - file-downloader@1:
+          inputs:
+            - destination: "$KEYSTORE_FILE"
+            - source: "$BITRISEIO_ANDROID_KEYSTORE_URL"
+
+      - install-missing-android-tools@2:
+          inputs:
+            - gradlew_path: "$PROJECT_LOCATION/gradlew"
+
+      # Execute Lint check
+      - android-lint@0:
+          inputs:
+            - project_location: "$PROJECT_LOCATION"
+
+      # Execute UnitTest
+      - android-unit-test@1:
+          inputs:
+            - module: "$MODULE"
+            - variant: "$VARIANT"
+            - project_location: "$PROJECT_LOCATION"
+
+      # Execute build on $VARIANT
+      - android-build@0:
+          inputs:
+            - module: "$MODULE"
+            - variant: "$VARIANT"
+            - project_location: "$PROJECT_LOCATION"
+
+      # Upload app-debug.apk to Apps & Artifacts for later downloading
+      - deploy-to-bitrise-io@1:
+          inputs:
+            - zip_name: ''
+            - deploy_path: /bitrise/deploy/app-debug.apk
+            - notify_email_list: ''
+          is_always_run: false
+
```

Comment:
I think we still need to archive step result on develop, release or master, not only primary 🤔

Reply:
@luongvo I updated to add the step on develop and release. We could ignore on master. How do you think?